frontend/src/app/utils/es-query.ts

```diff
@@ -10,6 +10,8 @@ import { findByName } from './utils';
 import { SearchFilter } from '../models/field-filter';
 import { APIQuery } from '../models/search-requests';
 import { TagFilter } from '../models/tag-filter';
+import { PageResultsParameters } from '../models/page-results';
+import { DeepPartial } from 'chart.js/dist/types/utils';
```

Comment:
Is this import from `chart.js` correct?

Reply:
It's not related to charts, but it's just a bit of laziness to import a `DeepPartial` typedef from there rather than define it ourselves.